test/dataUtils.spec.ts

```diff
@@ -245,5 +245,77 @@ describe('dataUtils', () => {
 
       expect(fooCount).to.equal(4);
     });
+
+    it('should replace all items, each with provided wrapper element', () => {
+      const compiled: any = new Function(
+        `return ${Handlebars.precompile(
+          fs.readFileSync(path.resolve(__dirname, './mock/foo.hbs'), 'utf-8'),
+        )}`,
+      )();
+      const template = Handlebars.template(compiled);
+
+      const mountSpy = spy();
+      const adoptSpy = spy();
+      const destructSpy = spy();
+
+      const foo = class Foo {
+        static displayName: string = 'foo';
+
+        constructor() {
+          // dom ready
+          mountSpy('foo');
+        }
+
+        adopted() {
```

Comment:
I think we can remove the adopted and dispose spies for this specific test. The mount spy could still be useful to check if the component is successfully created.

Reply:
Removing the dispose spy gives the following error when the test is run:
```
TypeError: removedComponent.instance.dispose is not a function
```

Reply:
The dispose function could stay, the spy can go away.

But maybe better to do a check on this line and only call dispose if it's available:
https://github.com/mediamonks/muban-core/blob/4159a16b2156bb006c9577162a40232801166824/src/lib/utils/cleanElement.ts#L21
(unrelated to this PR, but a nice quick fix to make these tests a bit shorter :))